src/options_parser.cc

```diff
@@ -30,6 +32,8 @@ OptionsResult* protagonist::ParseOptionsObject(Local<Object> optionsObject, bool
     optionsResult->serializeOptions.format = DRAFTER_SERIALIZE_JSON;
     optionsResult->serializeOptions.sourcemap = false;
     optionsResult->parseOptions.requireBlueprintName = false;
+    optionsResult->parseOptions.generateMessageBody = false;
```

Comment:
These new options should default to true, only when explicitly set by user to `false` they should be disabled. See the design at https://github.com/apiaryio/api-elements.js/pull/281#issuecomment-510881443

Reply:
Well, since we are defaulting them to `false` in drafter, we should do the same here. Otherwise, it will be inconsistent across our own libraries.